external/vcm/pyproject.toml

```diff
@@ -1,4 +1,4 @@
 # credit: https://stackoverflow.com/questions/60045913/installing-numpy-before-using-numpy-distutils-core-setup
 [build-system]
 build-backend = "setuptools.build_meta"
-requires=["numpy==1.19.4", "setuptools"]
\ No newline at end of file
+requires=["numpy==1.19.4", "setuptools==49.6.0"]
```

Comment:
Let's keep this change separate for a clean git history.

Reply:
Sure this is already merged, just had it to get the tests to pass